### PR TITLE
EOS-25506: LC : remove m0provision config from rpm spec files

### PR DESCRIPTION
**Pull request by atulsdeshmukh2312**
Signed-off-by: Atul Deshmukh <atul.deshmukh@seagate.com>

# Problem Statement
During image creation, it applied m0provision config stage which modifies the sysconfig parameter during image creation which is not required, m0provision config was required for R1 to apply the changes when we update the RPM's, with R2 and LC we already have mini provisioner which does the same job this is a redundant and can lead to errors in some environement so better to remove it from rpm spec file to apply config changes.

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: cortx-motr.spec.in [hunk 1]
```diff
@@ -345,7 +345,6 @@ getent passwd motr >/dev/null || useradd --system --shell /sbin/nologin \
 /sbin/depmod -a
 systemctl daemon-reload
 echo $(date) &> /var/log/motr-rpm-conf-update.log
```
Comment (madhavemuri) [inline cortx-motr.spec.in:347]: Can you remove this line as well ?

Comment (madhavemuri) [reply cortx-motr.spec.in:347]: @atulsdeshmukh2312 : Can you fix  it ?

Comment (atulsdeshmukh2312) [reply cortx-motr.spec.in]: Done fcd00f7

**Comment by mehjoshi** [comment]
@atulsdeshmukh2312 , Please update the checklist in the Testing section of the PR template.

**Review by madhavemuri — APPROVED** [review]

**Pull request merged**